### PR TITLE
Update TD Direct Investing

### PR DESCRIPTION
**Pull request by ghost**
updated sms support for TD direct investing - updated to market name

### CODE REVIEW

**Comment by mxxcon** [comment]
So the link that you mentioned in your other issue https://www.td.com/privacy-and-security/privacy-and-security/how-we-protect-you/online-security/2fa.jsp we use for personal banking.
Is TD Direct Investing uses the same login system as personal banking accounts? Becuase we are using this link as docs for banking entries.
Additionally, that link states that it supports phone calls in addition to sms, so if this PR were to be accepted, please add that info as well.

File: _data/investing.yml [hunk 1]
```diff
@@ -176,11 +176,12 @@ websites:
       img: tdameritrade.png
       tfa: No
 
-    - name: TD Waterhouse
+    - name: TD Direct Investing
       url: https://www.tdwaterhouse.ca
       twitter: TD_Canada
       img: tdameritrade.png
       tfa: No
```
Comment (mxxcon) [inline _data/investing.yml]: If `sms: yes` then `tfa` must be `yes` too.

**Review by mxxcon — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
Yes it uses the same system for both

**Review by ghost — COMMENTED** [review]
added phone and sms

**Comment by mxxcon** [comment]
What about https://authentication.td.com/uap-ui/index.html?consumer=easyweb&locale=en_CA#/login/easyweb-getting-started 
vs 
https://wb.authentication.td.com/uap-ui/index.html?consumer=webbroker&locale=en_CA#/login/webbroker-getting-started 
vs 
https://onlinebanking.tdbank.com/#/authentication/login ?
Seems like 3 different login system, no?

**Review by ghost — COMMENTED** [review]
What about https://authentication.td.com/uap-ui/index.html?consumer=easyweb&locale=en_CA#/login/easyweb-getting-started

^ Online Banking - with TD Canada trust ( Canadian) Once you sign in you can jump to your investing accounts) 

vs
https://wb.authentication.td.com/uap-ui/index.html?consumer=webbroker&locale=en_CA#/login/webbroker-getting-started

^ Direct access to "webroker" ( the name of their web trading platform) 
vs
https://onlinebanking.tdbank.com/#/authentication/login ?

^ TD Bank N.A The US bank that TD owns. 


Seems like 3 different login system, no?

^ 3 login portals they all use the same backend software/ offer the same features. TDbank.com is already updated correctly on here. 

To make this more confusing, TD Ameritrade is a separate entity in the USA that does not yet Support 2FA

—

File: _data/investing.yml [hunk 2]
```diff
@@ -176,11 +176,12 @@ websites:
...
 
-    - name: TD Waterhouse
+    - name: TD Direct Investing
       url: https://www.tdwaterhouse.ca
```
Comment (Carlgo11) [inline _data/investing.yml:180]:
```suggestion
       url: https://www.td.com/ca/en/investing/
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**